### PR TITLE
style-guide: mention tldr-lint instead of tldrl

### PR DESCRIPTION
**Pull request by marchersimon**
* I added the `--global` flag, because otherwise, it wouldn't be available as command.
* I replaced `tldrl` with `tldr-lint`, because with `tldr tldrl` you just get the alias page.

### CODE REVIEW

**Comment by marchersimon** [comment]
@blueskyson could you maybe translate the sentence I added for the Chinese style guide?

File: contributing-guides/style-guide.md [hunk 1]
```diff
@@ -27,12 +27,12 @@ It is run automatically on every pull request,
 but you may install it to test your contributions locally before submitting them:
 
 ```
-npm install tldr-lint
-tldrl -f {{page.md}}
+npm install --global tldr-lint
+tldr-lint -f {{page.md}}
```
Comment (nicokosi) [inline contributing-guides/style-guide.md:31]: Your command outputs the valid content in the standard output.
I think it is more useful to document the command that only displays errors:

```suggestion
tldr-lint {{page.md}}
```

See `tldr-lint --help`:
```
A linting tool to validate tldr pages

Options:
  -V, --version         output the version number
  -f, --format          also attempt formatting (to stdout, or as specified by -o)
  -o, --output <file>   output to formatted file
  -i, --in-place        formats in place
  -t, --tabular         format errors in a tabular format
  -v, --verbose         print verbose output
  -i, --ignore <codes>  ignore comma separated tldr-lint error codes (e.g. "TLDR001,TLDR0014")
  -h, --help            display help for command
```

**Review by nicokosi — APPROVED** [review]
:+1:

**Review by reinhart1010 — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]
